test/common/odp_cunit_common.c

```diff
@@ -273,6 +273,40 @@ static int default_term_func(void)
 	return odp_cunit_print_inactive();
 }
 
+static void _cunit_test_setup_func(void)
```

Comment:
Marvell copyright could be added to this file.

There is also a typo in the commit message: `inidividual `.